ffffm-button-bind/files/etc/rc.button/rfkill.btnb

```diff
@@ -2,27 +2,47 @@
 
 BIND=$(uci get button-bind.wifi.function)
 
+TMP=/tmp/wifi-button-status
```

Comment:
@rubo77 
Ja, das wäre ein gute Idee. Wegen Namespacing etc. evtl. `BTNB_WIFI_BUTTON_STATE` 
Kann das noch irgendwie in diesen PR mit eingebaut werden, oder müssen wir das nachträglich machen?